ietf-microwave-radio-link.yang

```diff
@@ -112,15 +113,6 @@ module ietf-microwave-radio-link {
    * Typedefs
    */
 
-  typedef power {
-    type decimal64 {
-      fraction-digits 1;
-    }
-    description
-      "Type used for the power values in the data nodes
-       for configuration or status.";
-  }
-
```

Comment:
Good idea to move the power data type definition to MW types module

However, this typedef was defined in RFC8561 so removing it will be an NBC change.

What about deprecating it?

Reply:
ok, will do this in a subsequent commit